packages/testing/package.json

```diff
@@ -38,7 +27,6 @@
   "files": [
     "lib",
     "src",
-    "generated-protos",
     "scripts"
```

Comment:
Can remove the `"scrips"` directory too, it doesn't exists anymore.

Reply:
True, thanks